playbooks/ps_80.yml

```diff
@@ -92,34 +92,17 @@
   - name: run Group Replication tests
     include_tasks: ../tasks/gr_test.yml
 
-  - name: copy the Tokudb config file on Debian/Ubuntu
-    template: src=../templates/tokudb.j2 dest=/etc/mysql/conf.d/tokudb.cnf
-    when: ansible_os_family == "Debian"
-
   - name: append include for RedHat
     lineinfile:
         path: /etc/my.cnf
         line: '!includedir /etc/my.cnf.d'
     when: ansible_os_family == "RedHat"
 
-  - name: copy the Tokudb config file on RHEL/CentOS/Amazon
-    template: src=../templates/tokudb.j2 dest=/etc/my.cnf.d/tokudb.cnf
-    when: ansible_os_family == "RedHat"
-
-  - name: install tokudb and restart server
-    command: /usr/bin/ps-admin --enable-tokudb
-
-  - name: restart mysql service
-    service: name=mysql state=restarted
-
-  - name: re-run ps-admin to finish tokudb install
-    command: /usr/bin/ps-admin --enable-tokudb
-
   - name: install percona-mysql-shell package
     include_tasks: ../tasks/install_pshell.yml
 
-  - name: check that Percona Server version is correct
-    command: /package-testing/version_check.sh ps80
+    #  - name: check that Percona Server version is correct
+    #    command: /package-testing/version_check.sh ps80
```

Comment:
Why do we remove version check step?

Reply:
Percona-mysql-shell is not updated yet for PS 8.0.28 , so in version check step it is throwing error while checking percona-mysql-shell version , so it is commented out till build team uploads latest percona-mysql-shell packages

Reply:
Done , executed one test for focal and it is also successful : https://ps80.cd.percona.com/job/package-testing-ps80-build/309/

Reply:
https://github.com/Percona-QA/package-testing/pull/147/commits/f699325459739a1c27814210a33c7359f41bea3b